src/components/transition/stories/FillMode.jsx

```diff
@@ -27,7 +27,7 @@ const shrinkFadeEffect = {
 };
 
 export const FillMode = () => {
-  const [active, setActive] = React.useState(false);
+  const [show, setShow] = React.useState(false);
```

Comment:
IMO, it should be `active` as it explains differences between fill modes when the `active` prop changes and it's not related to any widget or UI element like show/hide dialog.

Reply:
Probably you have an issue with button text since "active" and "inactive" aren't verbs and they don't match the current state in a delayed transition. I have the same issue with this example. I think we should leave `active/setActive` variables and change the button text to a verb related to activation.